tests/e2e/specs/register.spec.js

```diff
@@ -0,0 +1,64 @@
+import Data  from "../fixtures/register";
+const { firstName, lastName, email, password } = Data;
+const { url } = Cypress.env();
+
+describe("Register Page e2e test case", ()=>{
+    beforeEach(() => {
+        cy.visit('/')
+    })
+
+    it('visit app root url', ()=>{
+        cy.title().should('eq', 'vue-typescript')
+        cy.url('').should('eq', `${url}`);
+        cy.get('footer > a').should('have.attr','href', 'https://github.com/vindru/' )
```

Comment:
Please use data attributes to identify elements, see https://docs.cypress.io/guides/references/best-practices

Reply:
This is true for all test cases. The idea is our tests should not depend on DOM structure. It would make tests very hard to maintain.